app/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "app",
```

Comment:
you could improve ur app naming a lot....if you are to publish any package on npm (e.g. if it was a npm lib), it will be published as `app`

Reply:
it required me to put a folder name bfr running the React setup command so i kinda just forgot abt it lol